README.md

```diff
@@ -209,3 +209,12 @@ containers do not support. The template for creating custom content elements
 can be found [here](https://github.com/ExtensionEngine/tailor-element-template)
 while the template for creating custom containers can be found
 [here](https://github.com/ExtensionEngine/tailor-container-template).
+
+### Installing extensions
+
+1. copy (or git clone) extension files in `extensions/content-<elements or containers>/<my-extension-name>`
+2. create index.js file in `extensions/content-<elements or containers>` directory
+3. in the file from the previous step add `module.exports = ['my-extension-name'];`
```

Comment:
Convert this to code block so it gets proper syntax highlighting. :bowtie:

Reply:
@vladimyr please confirm this looks good now :)

Reply:
🙇